doc/utils/utils.py

```diff
@@ -0,0 +1,85 @@
+from os import PathLike
+from pathlib import Path
+from sphinx.cmd.build import get_parser
+from typing import Dict, Tuple, Optional
+
+
+ALL_DOCSETS = {
+    "nrf": "nRF Connect SDK",
+    "nrfxlib": "nrfxlib",
+    "zephyr": "Zephyr",
```

Comment:
This now says "Zephyr Project" in the doc set switcher when you're in the Zephyr doc set, but only "Zephyr" when you're in other doc sets. Should be consistent.

Reply:
Adjusted to `Zephyr Project`